static/src/javascripts/projects/common/modules/experiments/tests/giraffe.js

```diff
@@ -0,0 +1,63 @@
+define([
+    'common/utils/$',
+    'common/utils/template',
+    'text!common/views/giraffe-message.html',
+    'common/views/svgs'
+], function ($,template,giraffeMessage, svgs) {
+    return function () {
+
+        this.id = 'Giraffe';
+        this.start = '2016-07-03';
+        this.expiry = '2016-08-13';
+        this.author = 'Alex Ware';
```

Comment:
This author doesn't match the switch owner (you can add multiple people to the switch though, so maybe add Alex in there)